command/ca/init.go

```diff
@@ -411,6 +437,13 @@ func initAction(ctx *cli.Context) (err error) {
 			ui.Println()
 			return nil
 		}
+		// When initializing a linked CA, providing the --acme flag doesn't currently
+		// result in the default ACME provisioner being added. We may want to support this
+		// for ease of use, but this seems to require a bit of refactoring when generating
+		// the full CA configuration with DB initialization.
+		if deploymentType != pki.StandaloneDeployment && addDefaultACMEProvisioner {
+			return fmt.Errorf("adding a default ACME provisioner by providing the --acme flag is not supported with deployment type %q", deploymentType.String())
+		}
```

Comment:
Should we say something about using` step ca provisioner add acme --type ACME`?

Reply:
Might be good. This error will only be returned for linked deployments currently, and considering that we want to make it easy to do linked ACME, it's probably good to make this clear to the user, indeed.

Reply:
Fixed in d28911c85ea9cd4d4905bf94abeff523de6f9aae